lib/msgraph/models.go

```diff
@@ -83,10 +101,10 @@ type OptionalClaim struct {
 	// - netbios_domain_and_sam_account_name: netbiosDomainName\sAMAccountName
 	// - emit_as_roles
 	// - cloud_displayname
-	AdditionalProperties *[]string `json:"additionalProperties,omitempty"`
-	Essential            *bool     `json:"essential,omitempty"`
-	Name                 *string   `json:"name,omitempty"`
-	Source               *string   `json:"source,omitempty"`
+	AdditionalProperties []string `json:"additionalProperties,omitempty"`
+	Essential            *bool    `json:"essential,omitempty"`
+	Name                 *string  `json:"name,omitempty"`
+	Source               *string  `json:"source,omitempty"`
```

Comment:
nit: Godoc would be helpful here to understand what kind of data/value are we expecting for each field. For example, what could be the possible value for `Source`?